.github/workflows/registry_upload.yml

```diff
@@ -68,6 +71,8 @@ jobs:
           tags: |
             type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'main') }}
             type=ref,event=tag,enable={{is_default_branch}}
+            type=semver,pattern={{version}}
```

Comment:
Looking at the documentation, this is duplicate. Or actually, it wil tag with both `v0.2.0` and `0.2.0`.

Reply:
Looking at [this](https://github.com/docker/metadata-action?tab=readme-ov-file#latest-tag), we just need the semvar line, together with `flavor latest=auto`?

Reply:
Although that means if you tag a test branch, it will also be tagged latest, which is not good. So maybe this?

```
          tags: |
            type=raw,value=latest,enable={{is_default_branch}}
            type=semver,pattern={{version}}
```

Reply:
Hmm, `semver` would not trigger on a tag `test-123`, so that is not possible anyway. But it would build, just not get tagged? This is pretty complicated.

Reply:
I will change to only run on tags, from main it will only tag if the tag is a semver, if you a a commit on any other branch it will build images with that exact tag.